### DISCUSSION TITLE
Interested in using or maintaining this project?

### DISCUSSION THREAD

**Discussion by solebared** [rubyforgood/mutual-aid#1003] [General]
This project is in a period of diminution 〽️, and effectively on hiatus...
* ⚒️ We, the current maintainers, no longer have enough capacity to dedicate to it, and
* ♻️ Mutual aid efforts we'd been connected to have evolved. As of now, none of them are actively using the app.

We of course welcome any injection of energy in the form of new **maintainers, contributors** or interest in **using the app**. Please get in touch by commenting below.

**Comment by MutualAidz** [comment]
Hi there, there's interest in my area for investigating how different technologies could help in the creation and maintenance of mutual aid networks and supply chains, starting with a project to shift baby formula during the current shortage.  I'm new to git, and my task is to investigate what features this project has, including ease of use and visualization and accessibility.  Feel free to DM as well so I can go into detail.  Cheers

**Reply by solebared** [reply]
Hi @MutualAidz , thanks for getting in touch.

Probably the easiest way (likely for you and certainly for us :), to get a sense of the app is to click around our [demo site](http://mutual-aid-demo.herokuapp.com/). You can find admin credentials in the [readme](https://github.com/rubyforgood/mutual-aid/blob/main/README.md).

If you take a look and have further questions, one of us can likely answer them.
Cheers.

**Comment by fergmastaflex** [comment]
Hello! I am interested in helping out! I am a DC-based engineer with 10 years of experience mainly in Rails. I had the idea to possibly build a mutual aid app, but I'm glad I did some googling first. :joy: Let me know if I can help out in any way.
